rotkehlchen/data_import/importers/binance.py

```diff
@@ -287,7 +360,12 @@ class BinanceDepositWithdrawEntry(BinanceSingleEntry):
     """This class processes Deposit and Withdraw actions
         which are AssetMovements in the internal representation"""
 
-    AVAILABLE_OPERATIONS: Final[tuple[str, ...]] = ('Deposit', 'Withdraw')  # type: ignore[misc]  # figure out how to mark final only in this class
+    AVAILABLE_OPERATIONS: Final[tuple[str, ...]] = (  # type: ignore[misc]  # figure out how to mark final only in this class
+        'Deposit',
+        'Buy Crypto',  # it's a buy, with regular the payment methods, that deposits crypto
```

Comment:
Hmm this doesn't make sense. Is it a typo? What do you mean by regular payment methods? Something like, you deposit FIAT and it immediately appears as crypto? So something like this?

```suggestion
        'Buy Crypto',  # it's a direct buy where you deposit fiat and get credited with crypto
```